CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.2.1...HEAD)
+## [Unreleased](https://github.com/fivetran/go-fivetran/compare/v0.2.3...HEAD)
+
+## [0.2.3](https://github.com/fivetran/go-fivetran/compare/v0.2.2...v0.2.3) - 2021-11-10
```

Comment:
I think this is a new feature and not a fix. What about version `0.3.0`?

Reply:
Ok, let it be so)